spec/factories/profile.rb

```diff
@@ -20,5 +20,34 @@
         association(:profile_os_minor_version, os_minor_version: os_minor_version)
       end
     end
+
+    trait :with_values do
+      transient do
+        value_count { 5 }
+      end
+
+      after(:create) do |profile, evaluator|
+        create_list(
+          :v2_value_definition,
+          evaluator.value_count,
+          security_guide: profile.security_guide
+        )
+        values = profile.security_guide.value_definitions.each_with_object({}) do |value, object|
+          object[value.ref_id] = SecureRandom.random_number(10)
+        end
+        profile.update!(value_overrides: values)
```

Comment:
Maybe don't set all the definition values from a security guide, just a subset...same as not all rules are in a profile.